src/plugins/intel_cpu/src/config.cpp

```diff
@@ -51,8 +51,15 @@ Config::Config() {
     }
 #endif
 
-    if (!mayiuse(avx512_core_bf16))
-        enforceBF16 = false;
+    if (mayiuse(avx512_core_bf16)) {
```

Comment:
Then why not to use `ov::element::f32` as safe default option for each CPU architecture: x86/ARM now and RV64 in future? This default sounds very logical for me since actual inference precision value might be only changed by user (by inference_precision and execution_mode properties handled in readProperties()).
Finally this logic is not aligned with initialization code inside readProperties() which confuses a lot.

Reply:
Agree, will change